models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/IntraJobInvariantDefn/IntraJobInvariantDefn.masl

```diff
@@ -42,10 +42,8 @@ begin
         if theSequenceDefns'length = 1 then
           theSequenceDefn := find_one theSequenceDefns ();
         else
-          logMessage := "AESequenceDC::IntraJobInvariantDefn.createIntraJobInvariantDefn : Invalid Job Definition for this intra-job invariant definition (type 4) = " & theJobDefn.jobName;
-          Logger::log(Logger::Error, "AESequenceDC", logMessage);
-	      eventContent := "FailureReason = Invalid Job Definition for this intra-job invariant definition" & " : JobName = " & theJobDefn.jobName;
-	      Reporting~>reportEvent(Logger::Error, "svdc_job_definition_failed", eventContent);                
+          logMessage := "svdc_job_definition_failed : FailureReason = Invalid Job Definition for this intra-job invariant definition" & " : JobName = " & theJobDefn.jobName;
+          Logger::log(Logger::Warning, "pv.sv.intrajobinvariantdefn.createintrajobinvariantdefn ", logMessage);
```

Comment:
Is the "(type 4)" part of the original message significant here? - need to double-check.